Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "avr-device"
-version = "0.2.3"
+version = "0.2.4"
```

Comment:
Please drop this change.  I will do a version bump when it's time for a release; this should not be part of a pull-request.